css/first-line/demos/first-line/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Пример стилизации первой буквы в тексте — ::first-letter — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      min-height: 100vh;
+      margin: 0;
+      padding: 50px;
+      background-color: #18191c;
+      color: #ffffff;
+      font-family: "Roboto", sans-serif;
+      font-size: 16px;
+      letter-spacing: 0.04rem;
+    }
+
+    p {
+      width: 400px;
+    }
+
+    p::first-line {
+      text-transform: uppercase;
+      text-decoration: underline;
+      font-weight: bold;
+      color: deepPink;
```

Comment:
```suggestion
      color: #2E9AFF;
```

Цвет раздела CSS для цветового акцента.